test/exporter-subtree.js

```diff
@@ -36,7 +36,7 @@ module.exports = (repo) => {
           fileEql(files[0], smallFile, done)
         })
       )
-    })
+    }).timeout(10 * 1000)
```

Comment:
This is a small nitpick but a useful one. We prefer to have the timeouts at the top of the test so that it is easy to find it.